legend-engine-executionPlan-execution/src/main/java/org/finos/legend/engine/plan/execution/nodes/ExecutionNodeExecutor.java

```diff
@@ -259,7 +260,17 @@ public Result visit(PureExpressionPlatformExecutionNode pureExpressionPlatformEx
 
                 AppliedFunction f = (AppliedFunction) pureExpressionPlatformExecutionNode.pure;
                 SerializationConfig config = f.parameters.size() == 3 ? (SerializationConfig) ((ClassInstance) f.parameters.get(2)).value : null;
-
+                if (config != null && config.dateTimeFormat != null)
+                {
+                    try
+                    {
+                        DateTimeFormatter.ofPattern(config.dateTimeFormat.replaceAll("\"","'"));
+                    }
+                    catch (IllegalArgumentException e)
+                    {
+                        throw new IllegalArgumentException(e + "\n" + config.dateTimeFormat + " is not a valid dateTime format in SerializationConfig");
+                    }
```

Comment:
lets avoid concatenating exception messages (ie `e + ""...`.  The best practice is to include the original exception as the cause of the new one:  `new Exception("your msg", e)`

This will ensure at least us on support  will get proper context of the original error.

Reply:
Makes sense, sure. Also, touchbased with Mauricio on this, we will add the validation on studio end